node_package/tests/react_on_rails.spec.js

```diff
@@ -0,0 +1,12 @@
+var test = require('tape');
```

Comment:
Never wrote JS testing before. Is that possible to use ES6 syntax for testing code? cc: @justin808